### ISSUE TITLE
rebalance plugin throws "unexpected keyword argument 'msatoshi'"

### ISSUE THREAD

**Issue by dazl** [fort-nix/nix-bitcoin#687]
was happening in 0.0.104 and also now in 0.0.105

[this upstream commit](https://github.com/lightningd/plugins/commit/83a80d134ecb2adc6647235d56195332e846f5cb) may fix it. (I am trying to figure out how to pull it in to test it)

```
$ lightning-cli rebalance -k outgoing_scid=[...] incoming_scid=[...] msatoshi=500000sat maxfeepercent=0.2  getroute_method=iterative
{
   "code": -32600,
   "message": "Error while processing rebalance: LightningRpc.getroute() got an unexpected keyword argument 'msatoshi'",
   "traceback": "Traceback (most recent call last):\n  File \"/nix/store/xkvrc8zck504maw2d0lw3gclzgl4wmq9-python3-3.11.8-env/lib/python3.11/site-packages/pyln/client/plugin.py\", line 646, in _dispatch_request\n    result = self._exec_func(method.func, request)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/nix/store/xkvrc8zck504maw2d0lw3gclzgl4wmq9-python3-3.11.8-env/lib/python3.11/site-packages/pyln/client/plugin.py\", line 626, in _exec_func\n    ret = func(*ba.args, **ba.kwargs)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/nix/store/94ys9lbjzqf31rj91pgrz4w8c8bflwy7-clightning-plugin-rebalance-ce078bb/rebalance.py\", line 416, in rebalance\n    return cleanup(label, payload, rpc_result, e)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/nix/store/94ys9lbjzqf31rj91pgrz4w8c8bflwy7-clightning-plugin-rebalance-ce078bb/rebalance.py\", line 131, in cleanup\n    raise error\n  File \"/nix/store/94ys9lbjzqf31rj91pgrz4w8c8bflwy7-clightning-plugin-rebalance-ce078bb/rebalance.py\", line 333, in rebalance\n    r = getroute(targetid=incoming_node_id,\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/nix/store/94ys9lbjzqf31rj91pgrz4w8c8bflwy7-clightning-plugin-rebalance-ce078bb/rebalance.py\", line 209, in getroute_iterative\n    return plugin.rpc.getroute(targetid,\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: LightningRpc.getroute() got an unexpected keyword argument 'msatoshi'\n"
}
```

**Comment by jonasnick** [comment]
Thanks, this is potentially fixed by #689

**Comment by dazl** [comment]
Thank you.

**Comment by jonasnick** [comment]
Should be fixed by nix-bitcoin release 0.0.106.

**Comment by dazl** [comment]
it is fixed yes, thank you
